client/blocks/product-purchase-features-list/index.jsx

```diff
@@ -303,10 +280,6 @@ export class ProductPurchaseFeaturesList extends Component {
 				<GoogleMyBusiness selectedSite={ selectedSite } />
 				<FindNewTheme selectedSite={ selectedSite } />
 
-				<BusinessOnboarding
```

Comment:
Should this be feature-flagged? `jetpack/concierge-sessions`? It sounds like an offering that may be toggled on and off…

cc @chaselivingston

Reply:
That's certainly a reasonable option, and if it's not too much trouble to implement I think that's worth doing.

Reply:
Voilà: #32773